src/arrayManipulation.hpp

```diff
@@ -296,7 +297,8 @@ void resize( T * const LVARRAY_RESTRICT ptr,
   {
     if( newSize - size > 0 )
     {
-      std::memset( reinterpret_cast< void * >( ptr + size ), 0, ( newSize - size ) * sizeof( T ) );
+      size_t const sizeDiff = integerConversion< size_t >( newSize - size );
```

Comment:
Not that it matters too much, but LvArray seems to be using `std::` prefix pretty consistently
 ```suggestion
      std::size_t const sizeDiff = integerConversion< std::size_t >( newSize - size );
```

Reply:
it matters. thanks